tests/Admin/Admin_Page_Tests.php

```diff
@@ -39,7 +39,8 @@ public function test_add_page() {
 		wp_set_current_user( $admin_user );
 		set_current_screen( 'dashboard' );
 
-		$page_hook    = $this->admin_page->add_page();
+		$page_hook = $this->admin_page->add_page();
+		$this->admin_page->admin_menu();
```

Comment:
This test change does not look right. Because you now call both methods, it will result in `add_page()` being called twice. The tests pass, but that's still not the correct logic flow.

Can you move the `$page_hook = $this->admin_page->add_page();` line right above the last assertion? This way the method is only called then, at which point all data used in assertions has already been recorded, and we only need the `$page_hook` variable in the last assertion anyway.